lbrynet/dht/distance.py

```diff
@@ -1,21 +1,23 @@
+from binascii import hexlify
+
 from lbrynet.dht import constants
 
 
-class Distance(object):
+class Distance:
     """Calculate the XOR result between two string variables.
 
     Frequently we re-use one of the points so as an optimization
-    we pre-calculate the long value of that point.
+    we pre-calculate the value of that point.
     """
 
     def __init__(self, key):
```

Comment:
What format are these incoming keys?  It looks odd to encode it as base16, then decode it from base16 to get an int.

Reply:
@jackrobison or @shyba can you guys answer this one?

Reply:
nice catch! they come in big endian unsigned bytes. This function makes them into hex, then converts hex to int. It was like that in py2, but there is a better readable way to do in py3, see:
https://github.com/lbryio/lbry/pull/1370/files#diff-3a31e4218bf33300ff7b84e484ef1c2bR140
I was solving those cases, but totally forgot that one.

Reply:
@lbrynaut, @shyba made the changes and they have been merged into this PR.